build.gradle

```diff
@@ -53,6 +54,14 @@ plugins {
   id 'com.diffplug.spotless' version "6.14.0" apply false
 }
 
+testing {
```

Comment:
Could you please run it with `unitTest` and `integrationTest`? Also, please make sure the junit 4 style tests still works?

Reply:
I will take a look